apps/communications/dialer/js/calls_handler.js

```diff
@@ -34,16 +36,12 @@ var CallsHandler = (function callsHandler() {
   /* === Setup === */
   function setup() {
     if (telephony) {
+      telephony.addEventListener('callschanged', onCallsChanged);
+      onCallsChanged(); // The handler is invoked explicitly to prevent races
+
       // Somehow the muted property appears to true after initialization.
       // Set it to false.
       telephony.muted = false;
-      // Once bug 1129882 and bug 1113086 are landed, the new audio channel
-      // service will manage all the channels from System app, and to call
-      // this api is let the audio channel service know the Callscreen app
-      // is using the telephony audio channel.
-      if (telephony.ownAudioChannel) {
-        telephony.ownAudioChannel();
-      }
```

Comment:
This is being called by the callscreen window, which is opened by the communications app as an attention window. I'll try to move the opening in the main window and see what happens there.